package.json

```diff
@@ -58,9 +58,9 @@
     "node-sass": "4.11.0",
     "prettier": "1.15.3",
     "prismjs": "1.15.0",
-    "react": "16.7.0",
-    "react-dom": "16.7.0",
-    "react-helmet": "5.2.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-helmet": "^5.2.1",
```

Comment:
Had to upgrade helmet as well [cause of this](https://github.com/nfl/react-helmet/issues/437).